orderer/consensus/hlmirbft/chain.go

```diff
@@ -534,23 +554,50 @@ func (c *Chain) Submit(req *orderer.SubmitRequest, sender uint64) error {
 		c.Metrics.ProposalFailures.Add(1)
 		return err
 	}
-
-	leadC := make(chan uint64, 1)
+	submitPayload := req.Payload.GetPayload()
+	isfowarded := checkForwardFlag(submitPayload)
+	if isfowarded {
```

Comment:
Maybe just a matter of preference but would be clearer if this if statement is instead the else flow of the !isForwarded if statement.